<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Registration</class>
 <widget class="QMainWindow" name="Registration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wydatnik - rejestracja</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;

font-size: 10px;


max-width: 115px;

max-height: 27px;
}

QLineEdit {
   
    background-color: rgb(255, 255, 241);
    selection-color: white;                        
    selection-background-color: rgb(191, 31, 127);
    border: 2px groove gray;
    border-radius: 10px;
    padding: 2px 4px;
}*/

QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 10px;
padding-left: 5px;
padding-right: 5px;

max-width: 110px;

max-height: 15px;
}

#Login{
	background: gray;
}
#centralwidget{

	background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>100</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>321</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_login">
       <property name="text">
        <string>login:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_login"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_haslo">
       <property name="text">
        <string>hasło:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_haslo"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_haslo2">
       <property name="text">
        <string>powtórz hasło:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_haslo2"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_email">
       <property name="text">
        <string>email:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="edit_email"/>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button_register">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Zarejestruj</string>
       </property>
       <property name="icon">
        <iconset resource="icons/ikony.qrc">
         <normaloff>:/icon/Check-icon.png</normaloff>:/icon/Check-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Anuluj</string>
       </property>
       <property name="icon">
        <iconset resource="icons/ikony.qrc">
         <normaloff>:/icon/Cancel-icon.png</normaloff>:/icon/Cancel-icon.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>130</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter|Qt::AlignTop</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icons/ikony.qrc"/>
 </resources>
 <connections/>
</ui>
